antarest/study/storage/variantstudy/variant_snapshot_generator.py

```diff
@@ -30,22 +29,17 @@ def __init__(
         self,
         command_factory: CommandFactory,
         study_factory: StudyFactory,
-        exporter_service: ExporterService,
     ):
         self.command_factory = command_factory
         self.study_factory = study_factory
-        self.exporter_service = exporter_service
 
     def generate_snapshot(
-        self, variant_study: VariantStudy, parent_study: Study
+        self, variant_study: VariantStudy
```

Comment:
now this method call implicitly expect that a base raw study is present in the snapshot path. 
at least add an assert to check that there is a study in the snapshot path and raise an exception informing the user (the developer) that a base rawstudy must be copied into the snapshot path for this method to work.

or maybe this class should be deleted (as it is useless now on its own) and the method moved to VariantStorageService as a private method.